runtime/ftplugin/checkhealth.vim

```diff
@@ -8,7 +8,7 @@ endif
 
 runtime! ftplugin/help.vim
 
-setlocal wrap breakindent linebreak
+setlocal wrap breakindent linebreak nolist
```

Comment:
Needs to be added to the `undo_ftplugin` dance below

Reply:
Sorry, of course. Added `list<` now to both.